src/js/consts/shipmentType.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { RiFlightTakeoffLine, RiInformationLine, RiShipLine, RiSuitcaseLine, RiTruckLine } from 'react-icons/all';
+
+const TRANSLATION_PREFIX = 'react.stockMovement.shipmentType';
+
+const ShipmentType = {
+  AIR: {
+    icon: <RiFlightTakeoffLine />,
+    messageId: `${TRANSLATION_PREFIX}.air.label`,
+    defaultMessage: 'Air',
+  },
+  LAND: {
+    icon: <RiTruckLine />,
+    messageId: `${TRANSLATION_PREFIX}.land.label`,
+    defaultMessage: 'Land',
+  },
+  SUITCASE: {
+    icon: <RiSuitcaseLine />,
+    messageId: `${TRANSLATION_PREFIX}.suitcase.label`,
+    defaultMessage: 'Suitcase',
+  },
+  SEA: {
+    icon: <RiShipLine />,
+    messageId: `${TRANSLATION_PREFIX}.sea.label`,
+    defaultMessage: 'Sea',
+  },
+  DEFAULT: {
+    icon: <RiInformationLine />,
+    messageId: `${TRANSLATION_PREFIX}.default.label`,
+    defaultMessage: 'Default',
+  },
+};
+
```

Comment:
fyi: I agreed with @alannadolny that it didn't have any sense anymore, since after refactoring the translations, I got rid of a few usages of those and currently I'm using this call only once, so it doesn't require a function.